### PR TITLE
Mentorship programme: Feedback card component #28

### PR DESCRIPTION
**Pull request by joanaBrit**
## Description

Created a feedback card for the Mentorship Program. 
This card will display reviews and feedback from both mentees and mentors, , highlighting the program's impact.
This fixes the open issue number #28 

## Type

- [ ] Bug Fix
- [✅] New Feature
- [ ] Code Refactor
- [ ] Documentation
- [ ] Other

## Related Issue

This fixes the open issue - Mentorship programme: Feedback card component #28

## Screenshots
Figma:
Screenshot 2024-08-09 133354

Final:
Screenshot 2024-10-06 184718

⚠️The only difference is the name, mentee and year, It should be bold, but as in Figma shows a weight of 500px, where I applied but doesn't change, so how you want me to address to this?

## Testing

 On component level: Ensure you have a unit test in place. 

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [✅] I checked and followed the [contributor guide](../CONTRIBUTING.md)
- [✅] I have tested my changes locally.
- [✅] I have added a screenshot from the website after I tested it locally

### CODE REVIEW

**Comment by joanaBrit** [comment]
Just making sure this is seen -⚠️The only difference is the name, mentee and year, It should be bold, but as in Figma shows a weight of 500px, where I applied but doesn't change, so how you want me to address to this? @stepsen89  @lilyndk

**Review by dricazenck — APPROVED** [review]

File: src/components/FeedbackCard.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import { Box, Typography } from '@mui/material';
+import React from 'react';
+
+import QuoteIcon from '../../public/icons/quote-icon-custom_orange.svg';
```
[the remark this answers is not in the document]
Comment (joanaBrit) [reply src/components/FeedbackCard.tsx:4]: I’ve talked with @Lily, and I explained that I was following the path shown in Figma, which didn’t have any changes in the browser. She then advised me on using this icon. I know it’s not the best practice. What do you suggest I do?

[the remark this answers is not in the document]
Comment (dricazenck) [reply src/components/FeedbackCard.tsx:4]: My question was more about the navigation to get to the file: ../../ I was wondering if you could not use a variable/const for that

**Review by dricazenck — APPROVED** [review]

File: src/components/FeedbackCard.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (joanaBrit) [reply src/components/FeedbackCard.tsx]: Done it!

**Pull request merged**